pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -2473,3 +2482,236 @@ func virtioNicRequested(interfaces []v1.Interface) bool {
 
 	return false
 }
+
+// validateScheduler is function that validate spec.affinity
+// instead of bring in the whole kubernetes lib we simply copy it from kubernetes/pkg/apis/core/validation/validation.go
+// and vendor/k8s.io.apimachinery has to be updated because it's not updated to the latest version of the tag v0.23.5
+// which will leads to missing directory k8s.io/apimachinery/pkg/apis/meta/v1/validation eventually will cause we are unable
+// to copy validateAffinity function from kubernetes/pkg/apis/core/validation/validation.go
+func validateScheduler(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) (causes []metav1.StatusCause) {
```

Comment:
If we decide to go with copying, would you mind having all copied code in a dedicated file? https://github.com/kubevirt/kubevirt/blob/main/pkg/controller/expectations.go#L18 may be a good precedence.

If possible let's just keep the whole file pretty much unmodified to make detecting changes easy and future copying.

Reply:
@rmohr will do, but I still having a hard time on failure of "pull-kubevirt-verify-go-mod" test. Looks like  update vendor/k8s.io/apimachinery break down the test.  I remove the dir in `vendor/k8s.io/apimachinery` and run `go mod vendor ` again to force a update and copy to vendor. Any ideal what did wrong here ? 

Thanks a lot :)

Reply:
> Looks like update vendor/k8s.io/apimachinery break down the test. I remove the dir in vendor/k8s.io/apimachinery and run go mod vendor again to force a update and copy to vendor. Any ideal what did wrong here ?

Hm, I think you should just manually bump in `go.mod` and in the go.mod files in `staging` all references to 0.23.5 and then run `make dep-sync`.

Reply:
I'll give a try.  Thanks